handlers/deploy.go

```diff
@@ -9,21 +9,41 @@ import (
 	"net"
 	"net/http"
 	"os"
-	"os/exec"
 	"path"
 	"strings"
 
 	"github.com/alexellis/faasd/pkg/service"
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/cio"
-	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/namespaces"
 	"github.com/containerd/containerd/oci"
+	gocni "github.com/containerd/go-cni"
+	"github.com/google/uuid"
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/openfaas/faas-provider/types"
 	"github.com/pkg/errors"
 )
 
+const (
+	// TODO: CNIBinDir and CNIConfDir should maybe be globally configurable?
+	// CNIBinDir describes the directory where the CNI binaries are stored
+	CNIBinDir = "/opt/cni/bin"
+	// CNIConfDir describes the directory where the CNI plugin's configuration is stored
+	CNIConfDir = "/etc/cni/net.d"
+	// netNSPathFmt gives the path to the a process network namespace, given the pid
+	NetNSPathFmt = "/proc/%d/ns/net"
+
+	// defaultCNIConfFilename is the vanity filename of Ignite's default CNI configuration file
+	DefaultCNIConfFilename = "10-openfaas.conflist"
```

Comment:
Do we create this if not present?

Reply:
No need for weave code. I've removed that "Ignite" Reference.

Reply:
A future, but imminent use-case is for list functions to query state in containerd instead of state in the in-memory map. The weave code allows us to iterate containers and tasks and find out the IP for each. Can the CNI code do that too?

Reply:
This comment was about the `10-openfaas.conflis` file, does it get created automatically?

Reply:
Yes, it's created in main.go

Reply:
Updated the config and made sure the correct one is loaded even if there are other plugin configs in the net.d dir.